base_layer/core/src/transactions/transaction_components/wallet_output.rs

```diff
@@ -247,7 +247,7 @@ impl WalletOutput {
         ))
     }
 
-    pub async fn as_transaction_output<KM: TransactionKeyManagerInterface>(
+    pub async fn to_transaction_output<KM: TransactionKeyManagerInterface>(
```

Comment:
I dont agree with this change. 
`to_T` implies it takes `self`
`as_T` implies it takes `&self`

Reply:
nope

Reply:
`into_` - takes self
`to_` - usually `&self` some cost to the conversion
`as_` - usually `&self` cheap conversion

Reply:
Thanks, I stand corrected.

Reply:
TIL!